packages/react-components/src/AccessibilityScenarios/SplitButton.stories.tsx

```diff
@@ -0,0 +1,110 @@
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Button, SplitButton } from '@fluentui/react-button';
+
+import { Scenario } from './utils';
+
+export const EventReminderAccessibilityScenario: React.FunctionComponent = () => {
+  const [statusText, setStatusText] = React.useState<string | undefined>(undefined);
+  const statusRef = React.useRef<HTMLParagraphElement>(null);
+
+  const focusStatus = () => {
+    statusRef.current!.focus();
+  };
+
+  const onDismissButtonClick = () => {
+    setStatusText('The reminder has been dismissed.');
+    focusStatus();
+  };
+
+  const showPostponeText = (duration: string) => {
+    const postponeMessages: Record<string, string> = {
+      tenMinutes: 'for 10 minutes',
+      thirtyMinutes: 'for 30 minutes',
+      oneHour: 'for 1 hour',
+      tomorrow: 'until tomorrow',
+    };
+    setStatusText(`The event has been postponed ${postponeMessages[duration]}.`);
+    focusStatus();
+  };
+
+  return (
+    <Scenario pageTitle="Event reminder split button">
+      {!statusText && (
+        <>
+          <p>Your meeting starts in 10 minutes.</p>
+          <Button onClick={onDismissButtonClick}>Dismiss</Button>
+          <Menu positioning="below-end">
+            <MenuTrigger>
+              {(menuButtonProps: MenuButtonProps) => {
+                const extendedMenuButtonProps = {
+                  ...menuButtonProps,
+                  'aria-label': 'Postpone for later',
+                };
+
+                return (
+                  <SplitButton
+                    menuButton={extendedMenuButtonProps}
+                    primaryActionButton={{
+                      children: 'Postpone for 10 minutes',
+                      onClick: () => {
+                        showPostponeText('tenMinutes');
+                      },
+                    }}
+                  />
+                );
+              }}
+            </MenuTrigger>
+
+            <MenuPopover>
+              <MenuList>
+                <MenuItem
+                  onClick={() => {
+                    showPostponeText('thirtyMinutes');
+                  }}
+                >
+                  Postpone for 30 minutes
+                </MenuItem>
+                <MenuItem
+                  onClick={() => {
+                    showPostponeText('oneHour');
+                  }}
+                >
+                  Postpone for 1 hour
+                </MenuItem>
+                <MenuItem
+                  onClick={() => {
+                    showPostponeText('tomorrow');
+                  }}
+                >
+                  Postpone until tomorrow
+                </MenuItem>
+              </MenuList>
+            </MenuPopover>
+          </Menu>
+        </>
+      )}
+      <div ref={statusRef} tabIndex={0}>
+        {statusText && <p>{statusText}</p>}
+      </div>
+    </Scenario>
```

Comment:
Would we be able to add a splitbutton within a menu, either in this scenario or in the Menu scenario?

Reply:
Yes, I suppose. Once the SplitButton within Menu is implemented for v9. it's currently work in progress.

Reply:
So the split item in menu has been added.